app/services/text_ordinalizer.rb

```diff
@@ -0,0 +1,18 @@
+class TextOrdinalizer
+  def self.call(value)
+    text_ordinalize(value)
+  end
+
+  class << self
+  private
+
+    def text_ordinalize(value)
+      ordinalize_mapping[value] || value.ordinalize
+    end
+
+    def ordinalize_mapping
```

Comment:
Maybe this should be a constant rather than a method?

Reply:
You could argue that it should be in the locale yml files but based on the conversation we had the other week I think we are best off avoiding that.

Reply:
Have constantised it 👍